### PR TITLE
fix: add event bus name to cloudwatch event targets.

### PR DESCRIPTION
**Pull request by elliottohara**
Event targets need cloudwatch event bus name if they're non default. This PR simply adds it. 
Fixes https://github.com/moritzzimmer/terraform-aws-lambda/issues/93

I've tested this locally against a lambda with triggers against a custom bus and it works as expected.

### CODE REVIEW

**Comment by elliottohara** [comment]
Hello @moritzzimmer ... any way I can get this reviewed. It's pretty minor and our company uses your module a lot (thanks for it btw!)... We actually have a use case that needs to trigger lambdas off rules on the non default bus. 

Thanks!

File: cloudwatch_event_rules.tf [hunk 1]
```diff
@@ -23,9 +23,9 @@ resource "aws_cloudwatch_event_rule" "lambda" {
 }
 
 resource "aws_cloudwatch_event_target" "lambda" {
-  for_each = var.cloudwatch_event_rules
-
-  arn   = lookup(each.value, "cloudwatch_event_target_arn", local.function_arn)
-  rule  = aws_cloudwatch_event_rule.lambda[each.key].name
-  input = lookup(each.value, "cloudwatch_event_target_input", null)
+  for_each       = var.cloudwatch_event_rules
+  event_bus_name = lookup(each.value, "event_bus_name", null)
```
Comment (moritzzimmer) [inline cloudwatch_event_rules.tf:27]: please add a blank line after the `for_each`

**Review by moritzzimmer — CHANGES_REQUESTED** [review]
Thx for the PR @elliottohara. I have just one minor remark

File: cloudwatch_event_rules.tf (hunk 1 above)
Comment (elliottohara) [reply cloudwatch_event_rules.tf:28]: Done!

**Comment by moritzzimmer** [comment]
@elliottohara could you please fmt your changes/run pre-commit

**Comment by elliottohara** [comment]
> @elliottohara could you please fmt your changes/run pre-commit

Done.

**Review by moritzzimmer — APPROVED** [review]

**Comment by elliottohara** [comment]
Hi @moritzzimmer ... Any way we can get this merged and released? I'd love to bump our companies modules up to this.

Thanks!

**Comment by moritzzimmer** [comment]
@elliottohara this has been released with `v6.12.0` - thx for your contribution

**Pull request merged**